people.md

```diff
@@ -8,7 +8,9 @@ Thomas Caswell [@tacaswell](https://github.com/tacaswell)
 
 These are the current deputy project leader positions:
 
-- **Release Managers** :  (3.3-3.8) : Elliott Sales de Andrade [@qulogic](https://github.com/qulogic)
+- **Release Managers** :
+  - 3.3-3.9 : Elliott Sales de Andrade [@qulogic](https://github.com/qulogic)
+  - 3.10 : Kyle Sunden [@ksunden](https://github.com/ksunden)
```

Comment:
Kyle is actually 3.8/3.10/alternating (depending on funding situations how long that lasts)